Sources/General/Commands/Setup.swift

```diff
@@ -0,0 +1,258 @@
+//
+//  Copyright © 2020 Rosberry. All rights reserved.
+//
+
+import Foundation
+import ArgumentParser
+import ZIPFoundation
+
+final class Setup: ParsableCommand {
+
+    enum Error: Swift.Error {
+        case githubName(_ github: String)
+        case url(_ url: String)
+        case download(_ url: URL)
+        case write(_ url: URL)
+        case remove(_ url: URL)
+    }
+
+    static let configuration: CommandConfiguration = .init(commandName: "setup", abstract: "Provides your environment with templates")
+
+    @Option(name: .shortAndLong,
+            default: Constants.defaultTemplatesGithub,
```

Comment:
Such method for default values is deprecated
> Use regular property initialization for default values (`var foo: String = \"bar\"`)

Reply:
unfortunately, it does not works like this on our forks 
<img src ="https://user-images.githubusercontent.com/15152385/90503197-d8511280-e170-11ea-80b7-3d5425945d49.png" width="250"/>

Reply:
I think we should find out what's happening with ArgumentParser version

Reply:
Updated up to next major